src/Page/Index.elm

```diff
@@ -92,7 +93,11 @@ viewWhoWeAreList profiles =
     List.map
         (\profile ->
             li [ class "profile-item" ]
-                [ a [ href ("/about-us#" ++ generateId profile.name), class "profile-link" ]
+                [ a
+                    [ href ("/about-us#" ++ generateId profile.name)
+                    , class "profile-link"
+                    , Html.Styled.Attributes.target "_self"
```

Comment:
Do you need the full path if it's already imported from the module above?

Reply:
Maybe not. Don't have time to look now - but we probably shouldn't hard code it in there either. I'm happy for you to merge like this if you want to get on with styles.